reframe/core/schedulers/slurm.py

```diff
@@ -231,19 +231,19 @@ def filternodes(self, job, nodes):
         # Collect options that restrict node selection
         options = job.sched_access + job.options
         if job.sched_partition:
-            options.append('--partition=%s' % job.sched_partition)
+            options += ['--partition=%s' % job.sched_partition]
```

Comment:
```suggestion
            options += [f'--partition={job.sched_partition}']
```
Should you change this to use f-strings?